<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialogOptions</class>
 <widget class="QDialog" name="UDialogOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Налаштування ПК &quot;ТКЕнерго менеджер&quot;</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_rozdily">
       <property name="title">
        <string>Розділи</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_tabelOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Налаштування табеля</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_narahuvannyaOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Нарахування</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_utrymannyaOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Утримання</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_scriptsOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Скрипти нарахувань</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_opzOptions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>OPZ</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_Options">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_tabelOptions">
        <attribute name="title">
         <string>Табель</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_tabelVyd">
             <property name="title">
              <string>Спосіб враховування табеля</string>
             </property>
             <layout class="QVBoxLayout" name="_2">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioButton_fullTabel">
                <property name="text">
                 <string>Повний табель, з внесенням усіх днів місяця</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_SumTabel">
                <property name="text">
                 <string>Підсумковий табель, з внесенням загальних сум робочих днів і годин за місяць</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_narahOptions">
        <attribute name="title">
         <string>Нарахування</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_narahIndexaciya">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>337</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Індексація</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_noIndexaciya">
            <property name="text">
             <string>0. Не проводити індексацію</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_indexesNoNarah">
            <property name="text">
             <string>1. Обчислення/облік індексів без нарахування по індексації</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_doIndexaciya">
            <property name="text">
             <string>2. Проведення повноцінної індексації</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_utrymannya">
        <attribute name="title">
         <string>Утримання</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>231</width>
           <height>68</height>
          </rect>
         </property>
         <property name="title">
          <string>Визначення розміру соцпільги</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_socPilgaTilkyZOsobKartky">
            <property name="text">
             <string>Лише з особової картки працівника</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_socpilgaPoNarahuvanni">
            <property name="text">
             <string>По розміру нарахування</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_narahScriptOptions">
        <attribute name="title">
         <string>Скрипти нарахувань</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Шляхи до скриптів</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="margin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_vhSaldoScriptPath">
              <property name="text">
               <string>Вхідне сальдо</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_vhSaldoScriptPath"/>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="toolButton_vhSaldoScriptPath">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_osnNarahScriptPath">
              <property name="text">
               <string>Основне нарахування</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_osnNarahScriptPath"/>
            </item>
            <item row="1" column="2">
             <widget class="QToolButton" name="toolButton_osnNarahScriptPath">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_utrVidr01ScriptPath">
              <property name="text">
               <string>Відрахування і утримання ОД=01</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_utrVidr01ScriptPath"/>
            </item>
            <item row="2" column="2">
             <widget class="QToolButton" name="toolButton_utrVidr01ScriptPath">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_utrVidr02ScriptPath">
              <property name="text">
               <string>Відрахування і утримання ОД&gt;01</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_utrVidr02ScriptPath"/>
            </item>
            <item row="3" column="2">
             <widget class="QToolButton" name="toolButton_utrVidr02ScriptPath">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_oplataScriptPath">
              <property name="text">
               <string>Оплата</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_oplataScriptPath"/>
            </item>
            <item row="4" column="2">
             <widget class="QToolButton" name="toolButton_oplataScriptPath">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>213</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_opzOptions">
        <attribute name="title">
         <string>OPZ</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_vzaemodiya">
           <property name="title">
            <string>Взаємодія з програмою &quot;Податкова звітність&quot;</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_opzDirPath">
              <property name="text">
               <string>Шлях до папки програми &quot;Податкова звітність&quot;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineEdit_opzDirPath"/>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_opzDirPath">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_opzPidprId">
              <property name="text">
               <string>Вибір підприємства з програми &quot;Податкова звітність&quot;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_opzPidprId"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_programVersion_label">
              <property name="text">
               <string>Версія програми &quot;Податкова звітність&quot;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_programVersionValue">
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>277</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Прийняти</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_action_success-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/agt_action_success-256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Відхилити</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/button_cancel-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/button_cancel-256.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_zastosuvaty">
       <property name="text">
        <string>Застосувати</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
